docs/source/tags/datetime.md

```diff
@@ -0,0 +1,32 @@
+---
+title: DateTime
+type: tags
+order: 405
+is_new: t
+---
+
+The DateTime tag adds date and time selection to the labeling interface. Use this tag to add a date, timestamp, month, or year to an annotation.
```

Comment:
Should all tags be enclosed within ` (single quotes)?

Reply:
We don't do this in other tags, but I like this idea. This requires manual changing of every tag description, so I'll do it later.

Reply:
https://heartex.atlassian.net/browse/DEV-2876